src/it/dockerfile-docker-native-distroless/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/graalvm/native-image:ol7-java11-22.2.0 AS builder
+FROM ghcr.io/graalvm/native-image:ol7-java17-22.2.0 AS builder
```

Comment:
For consistency with the Gradle plugin, should update to `22.3.0`. We will likely upgrade to 23 before the 4.0.0 release though.